cmd/provisioner/hostpath-provisioner_test.go

```diff
@@ -72,7 +72,7 @@ func Test_isCorrectNode(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := isCorrectNode(tt.args.annotations, tt.args.nodeName); got != tt.want {
+			if got := isCorrectNode(tt.args.annotations, tt.args.nodeName, "kubevirt.io/provisionOnNode"); got != tt.want {
```

Comment:
Add a test for either annotation?

Reply:
Yes, this is why we put it on WIP, so we add some unit tests.